client/coral-framework/hocs/withQuery.js

```diff
@@ -1,7 +1,14 @@
 import * as React from 'react';
 import {graphql} from 'react-apollo';
 import {getQueryOptions, resolveFragments} from 'coral-framework/services/graphqlRegistry';
-import {getDefinitionName, separateDataAndRoot} from '../utils';
+import {getDefinitionName, separateDataAndRoot, getResponseErrors} from '../utils';
+
+const withSkipOnErrors = (reducer) => (prev, action, ...rest) => {
+  if (action.type === 'APOLLO_MUTATION_RESULT' && getResponseErrors(action.result)) {
```

Comment:
does apollo expose constants that we could import instead of using a string constant? probably not a big deal.